content/docs/reference-react.md

```diff
@@ -212,71 +210,71 @@ You will not typically invoke `React.createFactory()` directly if you are using
 React.isValidElement(object)
 ```
 
-Verifies the object is a React element. Returns `true` or `false`.
+檢查一個 object 是否為 React element。通常回傳 `true` 或是 `false`。
 
 * * *
 
 ### `React.Children` {#reactchildren}
 
-`React.Children` provides utilities for dealing with the `this.props.children` opaque data structure.
+`React.Children` 提供了一些工具可以將 `this.props.children` 作為不透明的資料結構處理。
 
 #### `React.Children.map` {#reactchildrenmap}
 
 ```javascript
 React.Children.map(children, function[(thisArg)])
 ```
 
-Invokes a function on every immediate child contained within `children` with `this` set to `thisArg`. If `children` is an array it will be traversed and the function will be called for each child in the array. If children is `null` or `undefined`, this method will return `null` or `undefined` rather than an array.
+對每一個列為 `children` 之中的直接 child 呼叫 function 並將 `this` 設定為 `thisArg`。如果 `children` 是一個 array，這將會列舉整個 array 並對每一個 child 呼叫這個 function。如果 children 是 `null` 或是 `undefined`，這個 method 將會回傳 `null` 或是 `undefined` 而不是一個 array。
```

Comment:
IMO `列舉` is more clearer than `拜訪` but it could just be my bias that I've seen a lot of translation associating `iteration` with `列舉` and it feel it's closer to `traversed` than `visit`.